jekyll/_cci2/build-processing.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: classic-docs
+title: "Enabling Build Processing"
+short-title: "Enabling Build Processing"
+description: "How to enable the preview of build processing"
+categories: [settings]
+order: 1
+---
+
+# CircleCI Build Processing
+
+This document describes how to enable the preview build processing engine if you need to trigger workflows from the CircleCI API.
+
+## Getting Started
+Enable build processing at the bottom of the Advanced section of Settings page for your project in the CircleCI app. If you don't see this setting, it should appear in the next few days for you as we roll it out to more and more organizations.
```

Comment:
We can now remove the "next few days" clause - it's up for everyone.

Reply:
removed